frontend/app/tests/e2e/pages/history-page/ledger-action-page.ts

```diff
@@ -20,6 +20,7 @@ export class LedgerActionPage {
     return () => {
       // Wait for response.status to be 200
       cy.wait('@apiCall').its('response.statusCode').should('equal', 200);
+      cy.wait(500);
```

Comment:
Is there not any alternative here other that hardcoding wait times?

Reply:
I'm not sure, I can't find other easy way for fixing this. The problem is sometimes, the order of flow is like this:
call api -> cypress get some element -> the api changes is refresh whole table -> the previous element that cypress get is gone now.

Reply:
> So @lukicenturi I found and issue. For the setup I have already a kraken exchange instance and tried to add a FTX one. Added the exchange and navigated to trades. Then I see two queries to the same endpoint with the same parameters
> 
> ![image](https://user-images.githubusercontent.com/5068010/147552535-c7be541a-e367-4583-822c-0e2347a1eab7.png)
> 
> And both query with `only_cache=true`. They both happen at the same time without me doing anything

Updated @yabirgb, can you try again?

Reply:
Can we allow this for now and think the other way later? @yabirgb